tests/test_ordinal_column.py

```diff
@@ -0,0 +1,161 @@
+"""
```

Comment:
I do some jiggery pokery with casting to force them to `int`s. You are right in that I need tests for that though